cf-create-org.sh

```diff
@@ -16,6 +16,13 @@ POP_END=$5
 MANAGER=$6
 
 MEMORY="${7:-4G}"
+MANAGER_ORIGIN="${8:-}"
```

Comment:
This means you have to set memory any time you want to set manager origin

Reply:
Took the default value out for `MEMORY`.